resources/js/components/catalog-new/PopoverFilter.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="tw-min-w-max md:tw-pb-2">
+        <button
+            class="tw-border tw-border-gray-300 tw-bg-white tw-py-3.5 tw-px-4 tw-text-lg tw-font-bold hover:tw-border-gray-800"
+            :class="{
+                'tw-border-gray-800': active,
+                'tw-border-gray-800 tw-border-2': popoverGroupControllerData.openedPopover === name,
```

Comment:
This logic is used 3 times here, so I'd probably move it it into a computed property
```suggestion
                'tw-border-gray-800 tw-border-2': isOpen,
```